### PR TITLE
HTMLref -> HTMLSidebar, part 3

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- Continuing https://github.com/mdn/content/pull/21956

### CODE REVIEW

File: files/en-us/web/html/element/s/index.md [hunk 1]
```diff
@@ -133,6 +133,6 @@ Some people who use screen readers deliberately disable announcing content that
 
 ## See also
 
-- The {{HTMLElement("strike")}} element, alter ego of the {{HTMLElement("s")}} element is obsolete and should not be used on Web sites any more.
+- The {{HTMLElement("strike")}} element, alter ego of the {{HTMLElement("s")}} element is obsolete and should not be used on websites anymore.
```
Comment (wbamberg) [inline files/en-us/web/html/element/s/index.md:136]:
```suggestion
- The {{HTMLElement("strike")}} element, alter ego of the {{HTMLElement("s")}} element is obsolete and should not be used on websites any more.
```

File: files/en-us/web/html/element/sub/index.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ This element only includes the [global attributes](/en-US/docs/Web/HTML/Global_a
 
 The `<sub>` element should be used only for typographical reasons—that is, to change the position of the text to comply with typographical conventions or standards, rather than solely for presentation or appearance purposes.
 
-For example, using `<sub>` to style the name of a company which uses altered baselines in their [wordmark](https://en.wikipedia.org/wiki/Wordmark) would not be appropriate; instead, CSS should be used (likely the {{cssxref("vertical-align")}} property, such as `vertical-align: sub` or, to more precisely control the baseline shift, `vertical-align: -25%`.
+For example, using `<sub>` to style the name of a company which uses altered baselines in their [wordmark](https://en.wikipedia.org/wiki/Wordmark) would not be appropriate; instead, CSS should be used (likely the {{cssxref("vertical-align")}} property), such as `vertical-align: sub` or, to more precisely control the baseline shift, `vertical-align: -25%`.
```
Comment (wbamberg) [inline files/en-us/web/html/element/sub/index.md:91]:
```suggestion
For example, using `<sub>` to style the name of a company which uses altered baselines in their [wordmark](https://en.wikipedia.org/wiki/Wordmark) would not be appropriate; instead, CSS should be used. For example, you could use the {{cssxref("vertical-align")}} property with a declaration like `vertical-align: sub` or, to more precisely control the baseline shift, `vertical-align: -25%`.
```

File: files/en-us/web/html/element/sub/index.md [hunk 3]
```diff
@@ -172,6 +172,6 @@ The output:
 
 ## See also
 
-- The {{HTMLElement("sup")}} HTML element that produces superscript. Note that you cannot use them both at the same time and you need to use [MathML](/en-US/docs/Web/MathML) to produce both a superscript directly above a subscript next to the chemical symbol of an element, representing its atomic number and its nuclear number.
+- The {{HTMLElement("sup")}} HTML element that produces superscript. Note that you cannot use them both at the same time, and you need to use [MathML](/en-US/docs/Web/MathML) to produce both a superscript directly above a subscript next to the chemical symbol of an element, representing its atomic number and its nuclear number.
```
Comment (wbamberg) [inline files/en-us/web/html/element/sub/index.md:175]:
```suggestion
- The {{HTMLElement("sup")}} HTML element that produces superscript. Note that you cannot use `sup` and `sub` both at the same time: you need to use [MathML](/en-US/docs/Web/MathML) to produce both a superscript directly above a subscript next to the chemical symbol of an element, representing its atomic number and its nuclear number.
```

File: files/en-us/web/html/element/sup/index.md [hunk 4]
```diff
@@ -163,6 +163,6 @@ The output:
 
 ## See also
 
-- The {{HTMLElement("sub")}} HTML element that produces subscripts. Note that you cannot use them both at the same time and you need to use [MathML](/en-US/docs/Web/MathML) to produce both a superscript and a subscript next to the chemical symbol of an element, representing its atomic number and its nuclear number.
+- The {{HTMLElement("sub")}} HTML element that produces subscripts. Note that you cannot use them both at the same time, and you need to use [MathML](/en-US/docs/Web/MathML) to produce both a superscript and a subscript next to the chemical symbol of an element, representing its atomic number and its nuclear number.
```
Comment (wbamberg) [inline files/en-us/web/html/element/sup/index.md:166]:
```suggestion
- The {{HTMLElement("sub")}} HTML element that produces subscripts. Note that you cannot use `sub` and `sup` at the same time: you need to use [MathML](/en-US/docs/Web/MathML) to produce both a superscript and a subscript next to the chemical symbol of an element, representing its atomic number and its nuclear number.
```

File: files/en-us/web/html/element/tbody/index.md [hunk 5]
```diff
@@ -120,8 +120,8 @@ This element includes the [global attributes](/en-US/docs/Web/HTML/Global_attrib
 ## Usage notes
 
 - If the table includes a {{HTMLElement("thead")}} block (to semantically identify a row of column headers), the `<tbody>` block _must_ come after it.
-- If {{HTMLElement("tr")}} elements are specified outside of an existing `<tbody>` element, as direct children of the {{HTMLElement("table")}}, these elements will be encapsulated by a separate `<tbody>` element generated by the browser.
-- When printing a document, the `<thead>` and {{htmlelement("tfoot")}} elements specify information that may be the same—or at least very similar—on every page of a multi-page table, while the `<tbody>` element's contents generally will differ from page to page.
+- If {{HTMLElement("tr")}} elements are specified outside an existing `<tbody>` element, as direct children of the {{HTMLElement("table")}}, these elements will be encapsulated by a separate `<tbody>` element generated by the browser.
+- When printing a document, the `<thead>` and {{htmlelement("tfoot")}} elements specify information that may be the same—or at least very similar—on every page of a multipage table, while the `<tbody>` element's contents generally will differ from page to page.
```
Comment (wbamberg) [inline files/en-us/web/html/element/tbody/index.md:124]: I can't decide if I like this better :).

**Review by wbamberg — COMMENTED** [review]
Just some copy-editing suggestions. Nice work as usual.

File: files/en-us/web/html/element/s/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/html/element/s/index.md]: 😕 https://www.merriam-webster.com/dictionary/anymore

Comment (wbamberg) [reply files/en-us/web/html/element/s/index.md]: gross, but I guess it is a US thing :)

File: files/en-us/web/html/element/tbody/index.md (hunk 5 above)
Comment (OnkarRuikar) [reply files/en-us/web/html/element/tbody/index.md:124]: https://www.merriam-webster.com/dictionary/multipage

**Review by wbamberg — APPROVED** [review]
🎉 great work!

**Pull request merged**